test/utilities/apexUtils.ts

```diff
@@ -18,26 +19,24 @@ export async function createApexClass(
   const workbench = await getWorkbench();
 
   // Using the Command palette, run SFDX: Create Apex Class to create the main class
-  const inputBox = await executeQuickPick('SFDX: Create Apex Class', Duration.seconds(1));
+  const inputBox = await executeQuickPick('SFDX: Create Apex Class', Duration.seconds(3)); // have to increase it otherwise it keeps failing on macos
 
   // Set the name of the new Apex Class
   await inputBox.setText(name);
-  await pause(Duration.seconds(1));
-  await browser.keys(['Enter']);
+  await inputBox.confirm();
   await pause(Duration.seconds(1));
 
   // Select the default directory (press Enter/Return).
-  await browser.keys(['Enter']);
-  await pause(Duration.seconds(1));
-
+  await inputBox.confirm();
+  await pause(Duration.seconds(2)); // have to hard pause for text editor to be ready
```

Comment:
What else can be used to wait for the new file to be ready for use?

Reply:
Great idea - I used `browswer.waitUntil`. Check out the function `checkFileOpen` I just implemented.